eslint.config.mjs

```diff
@@ -18,6 +18,14 @@ export default [
       },
     },
   },
+  {
+    files: ['**/*.test.js', '**/*.spec.js'],
```

Comment:
what's the difference between test.js and spec.js?

Reply:
might not need `*.spec.js` in our case, will remove. Thanks!